packages/core/src/components/pages/auth/components/updatePassword/index.tsx

```diff
@@ -1,44 +1,48 @@
 import React, { useState } from "react";
+import {
+    RefineUpdatePasswordFormTypes,
+    RefineUpdatePasswordPageProps,
+} from "@pankod/refine-ui-types";
 
-import { useTranslate, useRouterContext, useUpdatePassword } from "@hooks";
-import { IAuthCommonProps, IUpdatePasswordForm } from "../..";
+import { useTranslate, useUpdatePassword } from "@hooks";
 
-export const UpdatePassword: React.FC<IAuthCommonProps> = ({
-    backLink,
-    submitButton,
+import { DivPropsType, FormPropsType } from "../..";
+
+type UpdatePasswordProps = RefineUpdatePasswordPageProps<
+    DivPropsType,
+    DivPropsType,
+    FormPropsType
+>;
+
+export const UpdatePasswordPage: React.FC<UpdatePasswordProps> = ({
+    wrapperProps,
+    contentProps,
+    renderContent,
+    formProps,
 }) => {
     const translate = useTranslate();
-    const { Link } = useRouterContext();
 
-    const { mutate: updatePassword } = useUpdatePassword<IUpdatePasswordForm>();
+    const { mutate: updatePassword, isLoading } =
+        useUpdatePassword<RefineUpdatePasswordFormTypes>();
 
     const [newPassword, setNewPassword] = useState("");
     const [confirmPassword, setConfirmPassword] = useState("");
 
-    const renderLink = (link: React.ReactNode, text?: string) => {
-        if (link) {
-            if (typeof link === "string") {
-                return <Link to={link}>{text}</Link>;
-            }
-            return link;
-        }
-        return null;
-    };
-    return (
-        <div>
+    const content = (
+        <div {...contentProps}>
             <h1 style={{ textAlign: "center" }}>
                 {translate("pages.updatePassword.title", "Update Password")}
             </h1>
             <hr />
             <form
                 onSubmit={(e) => {
                     e.preventDefault();
-                    if (newPassword === confirmPassword) {
-                        updatePassword({
-                            newPassword,
-                        });
-                    }
+                    updatePassword({
+                        password: newPassword,
+                        confirmPassword,
+                    });
```

Comment:
```suggestion
                    updatePassword({
                        password: newPassword,
                    });
```
It looks like we don't need to send the `confirmPassword`

Reply:
I added it because we didn't verify that two passwords are the same in core. I think this can be check in authProvider.

Reply:
cool 🚀